CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 #### 0.2.1 (Next)
 
+* Add support for block_actions interaction type - [@CrazyOptimist](https://github.com/CrazyOptimist).
```

Comment:
Add a PR number in front:

```
[#6](https://github.com/slack-ruby/slack-ruby-bot-server-events/pull/6): Add ...
```

Reply:
Done
Thank you so much for quick response!